index.js

```diff
@@ -1,5 +1,3 @@
-'use strict'
```

Comment:
This is necessary

Reply:
Hello @Eomm ,
Done, I also adapted the previous test ...